### PR TITLE
Update UG for PE-D bug fixes and edit triage

### LINKED ISSUE #141: [PE-D][Tester A] Error thrown when only editing the Triage
* edit function fails when only editing the Triage.
* type `edit S1234567A t/4` to edit the triage for that NRIC. 

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: Feng1231/ped#6

### LINKED ISSUE #163: [PE-D][Tester D] Tags Should Allow Multiple Words
It seems that tags do not support multiple words. For example, 'tag/High Blood Pressure' triggers an error.
But High Blood Pressure is very common ill.

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: publicmain/ped#3

### LINKED ISSUE #180: [PE-D][Tester D] Didn't Handle Error about Email Format
It should allow me to update a person's email as .com.com.com.com which is invalid

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: publicmain/ped#10

### LINKED ISSUE #195: [PE-D][Tester C] Person view window still exists after deletion
After deletion of a person, if the person's detailed view is open before deletion, it will still remain on screen.

Before deletion:

After deletion:

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: jeanyang04/ped#11

### LINKED ISSUE #198: [PE-D][Tester C] View window does not update after edit
1. Viewing a person's details
2. Edit the person's details
3. Edit successful, but does not update in view window.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: jeanyang04/ped#14

### LINKED ISSUE #211: Confusing reference to patient identified by index number in UG
Screenshot 2024-11-11 at 1 04 11 PM

Technically not wrong, but probably should change to sth like "as specified by NRIC" instead of referring to index

### LINKED ISSUE #212: Update UG Help Window image

### LINKED ISSUE #214: Include in UG that appointment cant be added through add command

### PR DESCRIPTION
**Pull request by Dominic-Khoo**
CHANGELOG:
- Included note in UG that ViewWindow will not update automatically upon edit/deletion of patient until the window is closed and the `view` command is entered again. (Closes #195 , closes #198 )
- Included warning in UG that Murphy's List does not check for invalid email formats. (Closes #180 )
- Included warning in UG that tags must be alphanumeric and cannot contain whitespaces. (Closes #163 )
- Included note in UG that appointment cannot be added in `add` command (Closes #214 )
- Included note that find will only search by name or by tags but not both at the same time
- Added description for Appointment Popup into UG 
- Added description for Command Shortcuts into UG
- Fixed an issue where editing ONLY triage of a patient would result in an error message (Closes #141 )
- Updated HelpWindow and corresponding image in UG with appropriate fields (Closes #211 , closes #212 )

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -334,3 +358,23 @@ You can check your Java version by running `java -version` in the command termin
...
+| `exit`        | `ex`     | Exits the application                    |
+| `find`        | `f`      | Finds patients by name or tags           |
+| `help`        | `h`      | Displays a list of accepted commands     |
+| `list`        | `ls`     | Shows a list of all patient profiles     |
+| `remark`      | `r`      | Add a remark to a patient profile        |
+| `sort`        | `s`      | Sorts the list of patients               |
+| `triage`      | `t`      | Change triage stage of a patient         |
+| `view`        | `v`      | Views detailed patient information       |
+
+> **Note:** Shortcuts for `log`and `schedule` are not included but will be available in a future update.
```
Comment (volleyballkickedme) [inline docs/UserGuide.md:380]: LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -84,8 +86,13 @@ If you can type fast, Murphy's List can get your healthcare administrative tasks
     e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```
Comment (itsnotsherm) [inline docs/UserGuide.md:90]: nice inclusion to clarify functionality of tags field

**Review by volleyballkickedme — APPROVED** [review]
LGTM, ready to merge

File: docs/UserGuide.md [hunk 3]
```diff
@@ -108,19 +115,19 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com i/S1234123A a/John Street, Block 123, #01-01 t/2`
 * `add n/Betsy Crowe p/24681357 e/betsycrowe@example.com i/T1234567D a/Clementi Ave 1, Block 230 t/1 tag/Diabetic tag/G6PD`
 
+> **Note:** `add` **does not support** the addition of appointments. Use the [appointment](#adding-an-appointment--appointment) command to do so.
```
Comment (itsnotsherm) [inline docs/UserGuide.md:118]: good inclusion for this too

File: docs/UserGuide.md [hunk 4]
```diff
@@ -108,19 +115,19 @@ Examples:
...
+
 ### Adding a remark to a patient profile : `remark`
 
 Adds a remark to a specified patient profile.
-> 💡 **Note:** You can only specify **one** remark to add
+> **Note:** You can only specify **one** remark to add, and patients can only have one remark.
```
Comment (itsnotsherm) [inline docs/UserGuide.md:123]: maybe can add on that if remark command is executed by the user for a patient that already has a remark, the original will be overwritten

**Review by itsnotsherm — APPROVED** [review]
LGTM

**Pull request merged**